packages/auth/ios/RNFBAuth/RNFBAuthModule.m

```diff
@@ -954,6 +954,17 @@ - (NSDictionary *)getJSError:(NSError *)error {
   NSString *message = [error localizedDescription];
   NSString *nativeErrorMessage = [error localizedDescription];
 
+  if(error.domain == @"FIRAuthErrorDomain" && code == 17999){
```

Comment:
is code an actual integer or is it `NSNumber`? Also - might want to leave a note above the if statement explaining why the exception has its own if statement.